src/config.ts

```diff
@@ -158,6 +185,10 @@ export interface ExtensionConfigurationSettings {
     configureEnvironment: Environment;
     buildEnvironment: Environment;
     testEnvironment: Environment;
+    packEnvironment: Environment;
+    cpackPath: string;
+    cpackArgs: string[];
+    // no settings needed for workflow presets execution?
```

Comment:
Stale comment. I would agree that no settings are needed, since it actually calls into other presets execution stuff.